packages/react-components/react-search/stories/SearchBox/SearchBoxDefault.stories.tsx

```diff
@@ -1,4 +1,6 @@
 import * as React from 'react';
 import { SearchBox, SearchBoxProps } from '@fluentui/react-search';
 
-export const Default = (props: Partial<SearchBoxProps>) => <SearchBox {...props} />;
+import { FilterRegular } from '@fluentui/react-icons';
+
+export const Default = (props: Partial<SearchBoxProps>) => <SearchBox {...props} contentAfter={<FilterRegular />} />;
```

Comment:
At some point I think it'd be good to move the `<FilterRegular />` into a specific separate example to leave the Default example bare-bones, but that can wait until another PR for adding stories :)